### PR TITLE
Problem: there are no f-strings in python 3.5

### PR DESCRIPTION
**Pull request by codegeschrei**
Solution: change f-strings to a compatible format for python 3.5

### CODE REVIEW

File: bigchaindb/tendermint/lib.py [hunk 1]
```diff
@@ -162,7 +162,7 @@ def get_utxoset_merkle_root(self):
         # See common/transactions.py for details.
         hashes = [
             sha3_256(
-                f'''{utxo['transaction_id']}{utxo['output_index']}'''.encode()
+                '{}{}'.format(utxo['transaction_id'], utxo['output_index']).encode()
```
[the remark this answers is not in the document]
Comment (codegeschrei) [reply bigchaindb/tendermint/lib.py]: Yes!
With f-string you have the format of `f'x'`. When you now want a dict key, e.g. x['a'] you can't just write `f'x['a']'` that's why we used triple quotes (doubles are fine too)

[the remark this answers is not in the document]
Comment (ttmc) [reply bigchaindb/tendermint/lib.py]: Ah, that explains it. Thanks!

**Comment by vrde** [comment]
Great! :clap: 
Any plan to test against Python 3.5?

**Comment by codegeschrei** [comment]
We could add it to travis again?

**Comment by vrde** [comment]
> We could add it to travis again?

It makes sense, otherwise we won't be 100% sure we are compatible with 3.5

**Comment by vrde** [comment]
Can you update `setup.py` to include the following in the classifiers?

```
 'Programming Language :: Python :: 3.5',
```

**Pull request merged**